stdlib/Base64/src/decode.jl

```diff
@@ -209,6 +209,7 @@ julia> String(b)
 ```
 """
 function base64decode(s)
+    s = replace(s, r"\s" => "")
```

Comment:
This implementation is not ideal since it copies the data. Would be best to handle it during decoding. I also believe it is only necessary to handle ASCII space and newline.

Reply:
Also, by [RFC 2045](https://www.ietf.org/rfc/rfc2045.txt): "All line breaks or other characters not  found in Table 1 must be ignored by decoding software.".

Reply:
So maybe do a `|| (space) || (newline)` in here?
https://github.com/JuliaLang/julia/blob/b56a9f07948255dfbe804eef25bdbada06ec2a57/stdlib/Base64/src/decode.jl#L102

Reply:
Hi, I wrote the original code of this implementation to improve the performance based on my package (https://github.com/bicycle1885/CodecBase.jl). The implementation in this package ignores whitespace characters so it may help here. Otherwise, I can fix the issue instead of you if you are not in a rush.

Reply:
Thanks, I think I can pull this one off, Of course it would be easier if you can give me some hint on which part of the code is used to ignore invalid characters when decoding.

Reply:
I don't remember the code well, but I think you can modify the decoding behavior just by modifying its look-up table.

Reply:
I think that was the case in your CodecBase but not in Julia's code base.